x/treasury/genesis.go

```diff
@@ -10,25 +12,26 @@ import (
 func InitGenesis(ctx sdk.Context, keeper Keeper, data GenesisState) {
 	keeper.SetParams(ctx, data.Params)
 
-	for epoch, taxRate := range data.TaxRates {
-		keeper.SetTaxRate(ctx, int64(epoch), taxRate)
-	}
-
-	for epoch, rewardWeight := range data.RewardWeights {
-		keeper.SetRewardWeight(ctx, int64(epoch), rewardWeight)
-	}
+	keeper.SetTaxRate(ctx, data.TaxRate)
+	keeper.SetRewardWeight(ctx, data.RewardWeight)
+	keeper.SetEpochInitialIssuance(ctx, data.EpochInitialIssuance)
+	keeper.SetTaxProceeds(ctx, data.TaxProceed)
 
 	// store tax caps
 	for denom, taxCap := range data.TaxCaps {
 		keeper.SetTaxCap(ctx, denom, taxCap)
 	}
 
-	for epoch, historicalIssuance := range data.HistoricalIssuances {
-		keeper.SetHistoricalIssuance(ctx, int64(epoch), historicalIssuance)
-	}
+	fmt.Println(data.MRs)
```

Comment:
Remove print statement; in fact, let's do a global search to make sure we are not printing unecessary info